src/spatial_reaction_systems/lattice_reaction_systems.jl

```diff
@@ -8,7 +8,7 @@ const GridLattice{N, T} = Union{Array{Bool, N}, CartesianGridRej{N, T}}
 """
 $(TYPEDEF)
 
-A representation of a spatial system of chemical reactions on a discrete (lattice) space. 
```

Comment:
Suddenly all my PRs have started to remove random spaces from any files I touch. Nothing bad, but does clog up the PR changes. Have seen it with other PRs earlier as well.

Reply:
This is probably some VSCode setting you want to turn off.